DFC.ServiceTaxonomy.Editor/Recipes/01_SkillsToolkit_Pages.recipe.json

```diff
@@ -0,0 +1,334 @@
hen and how you can use copyright material (including material that is Crown copyright) including downloading it onto your device</li><li>when and how you must acknowledge the course provider or the original copyright owner</li><li>when and where you should include a reference to the Open Government Licence or any specific copyright statement</li></ul><h2 class=\"govuk-heading-l\">User terms: Accessing and using The Skills Toolkit</h2><p class=\"govuk-body-m\">The Skills Toolkit contains a list of online educational resources which has been compiled and is managed by the Department for Education (referred to as \"we\" or \"us\"). Users are referred to as \"you\" whether in\r\n        their individual capacity or as agents or representatives of an organisation they work for. Access to The Skills Toolkit list through this Website is subject to:</p><ul class=\"govuk-list govuk-list--bullet\"><li>the terms and conditions of use of the course providers set out on their websites, including any relevant disclaimers</li><li>the below The Skills Toolkit Disclaimer</li><li>GOV.UK's terms and conditions</li></ul><p class=\"govuk-body-m\">By accessing The Skills Toolkit, you agree to the terms and conditions of use of The Skills Toolkit, when you view or upload the homepage of The Skills Toolkit through the homepage of the main DfE website published on GOV.UK. When you enrol on any\r\n        of the online courses, you will be bound by the terms and conditions of use of the relevant course provider.</p><h2 class=\"govuk-heading-l\">The Skills Toolkit disclaimer</h2><p class=\"govuk-body-m\">Although we make every reasonable effort to select appropriate courses, we do not have any control over them and, in particular, we are not responsible for:</p><ul class=\"govuk-list govuk-list--bullet\"><li>the protection of any information you give to course providers</li><li>your failure to comply with relevant copyright or other intellectual property right law</li><li>any loss or damage that may come from any use or misuse of course provider websites, or any other websites they link to (together 'Third Party Websites')</li></ul><p class=\"govuk-body-m\">You must not assume that the list of courses on The Skills Toolkit is exhaustive, or that sources or resources that have been omitted are of no value or inappropriate as online educational resources.</p><p class=\"govuk-body-m\">You should read all terms and conditions, privacy\r\n        policies and end user licences that relate to the online courses before you enrol as you will be bound by their terms when you enrol.</p><p class=\"govuk-body-m\">You are responsible for choosing which online course for which you want to enrol and we will not have any responsibility if your selected online course does not meet your requirements or provide the learning you require. </p><p class=\"govuk-body-m\">You agree to release us from any claims or disputes that may arise from your use of Third Party Websites.</p></section></div>"
```

Comment:
Are the classes that start with `sf` Sitefinity classes?
If so, are they available when this gets displayed by the pages app?